### PR TITLE
Make things compile and consistent

### PR DESCRIPTION
**Pull request by sdroege**

### CODE REVIEW

File: gtk-examples/src/custom_cell_renderer.rs [hunk 1]
```diff
@@ -34,19 +35,23 @@ static PROPERTIES: [Property; 0] = [
 ];
 
 impl CellRendererCustom {
-    pub fn new() -> Self {
+    pub fn new() -> CellRenderer {
```
Comment (sdroege) [inline gtk-examples/src/custom_cell_renderer.rs]: If you want to go the gst-plugin-simple approach, you would here have to pass in something for the backing implementation and then register a new type once per backing implementation (the backing implementation is what would be stored in your CellRendererCustomStatic then and would be available from init, etc)

**Comment by vhdirk** [comment]
Thanks a lot!

**Comment by vhdirk** [comment]
Seeing how you return ```CellRenderer``` here, and not ```CellRendererCustom```, does answer my question regarding ```IsA``` somewhat. There would be no way to directly set members of ```CellRendererCustom``` or call members on it? Everything would have to go through property getters/setters? That may even be a good thing, given that it would be nice to be able to access this from other languages as well.

**Comment by sdroege** [comment]
You can get the `CellRendererCustom` with `get_impl`, but this was really only intended to be done from inside the implementation.

Currently there is no Rust wrapper for our custom type, which is your problem here. For GStreamer plugins that's not really required or even useful. But you could add one with `glib_wrapper!` and by making the type registration a separate function, and then you could add public API on the Rust wrapper for doing things directly instead of via property setters/getters.

**Pull request merged**